src/convertToObject.js

```diff
@@ -6,7 +6,24 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  // перетворити рядок на масив де елемент це правило css
+  // перевірити чи рядок не порожній
+
+  const listStyles = sourceString
+    .split(';')
+    .filter((string) => string.trim() !== '');
+
+  const stylesObject = {};
+
+  listStyles.forEach((entry) => {
+    const [key, value] = entry.split(':').map((el) => el.trim());
```

Comment:
The use of `.map` is appropriate here to trim both the key and value. This is a good use of array methods to process the string into an object.